tools/osbuilder/node-builder/azure-linux/common.sh

```diff
@@ -36,9 +38,12 @@ else
 	UVM_TOOLS_PATH_SRC="/opt/kata-containers/uvm/src"
 	UVM_PATH_DEFAULT="${INSTALL_PATH_PREFIX}/share/kata-containers"
 	IMG_FILE_NAME="kata-containers.img"
+	IMG_DBG_FILE_NAME="kata-containers-debug.img"
 	SHIM_CONFIG_PATH="${INSTALL_PATH_PREFIX}/share/defaults/kata-containers"
 	SHIM_CONFIG_FILE_NAME="configuration-clh.toml"
-	SHIM_CONFIG_INST_FILE_NAME="configuration.toml"
+	SHIM_CONFIG_INST_FILE_NAME="${SHIM_CONFIG_FILE_NAME}"
```

Comment:
I tink this should remain as is. Can discuss

Reply:
I am not sure I agree with the changes in this file here. Let's discuss explicitly what file names we want fo all image/config/IGVM files for PodSB, ConfPods debug / non-debug